Packs/CommonPlaybooks/Playbooks/playbook-CVE_Enrichment_-_Generic_v2.yml

```diff
@@ -97,15 +104,20 @@ tasks:
     note: false
     timertriggers: []
     ignoreworker: false
+    continueonerrortype: ""
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
   '4':
     id: '4'
-    taskid: 842e273c-40ee-46c1-837d-f7e09bcefef0
+    taskid: 5199c9bb-9a57-4d36-80fe-3bc1035e785e
     type: condition
     task:
-      id: 842e273c-40ee-46c1-837d-f7e09bcefef0
+      id: 5199c9bb-9a57-4d36-80fe-3bc1035e785e
       version: -1
       name: Is IBM X-Force Exchange integration enabled?
```

Comment:
no need for this integration check
seems like `cve-search` is also a generic command. mark it also as skip if unavailable